src/main/resources/bootstrap.yaml

```diff
@@ -3,9 +3,10 @@ spring:
     propertiesvolume:
       enabled: true
       prefixed: false
-      paths: /mnt/secrets/bulk-scan
+      paths: /mnt/secrets/bulk-scan,/mnt/secrets/s2s
```

Comment:
you should remove s2s vault dependency and keep it in `bulk-scan vault`

Reply:
but why are we avoiding this feature? i thought it's a excellent advantage

Reply:
because in AAT pipeline won't allow to read from other vaults. It needs a managed identity